docs/guides/taxes.rst

```diff
@@ -49,3 +57,35 @@ Fetching taxes
 
     By default, Saleor is making requests to the VatLayer API through HTTP (insecure),
     if you are using a paid VatLayer subscription, you may want to set the settings ``VATLAYER_USE_HTTPS`` to ``True``.
+
+
+Avalara
+-------
+
+All configuration and account setup should be done on the Avalara's admin panel. Saleor uses the standard API of
+Avalata's to fetch tax codes, to calculate taxes during the checkout process and to transfer data directly to the
+Avalara's system.
+
+Configuration
+^^^^^^^^^^^^^
+
+Only basic data needs to be provided on Saleor's side. You need to fill up the following variables:
+``AVATAX_USERNAME_OR_ACCOUNT``, ``AVATAX_PASSWORD_OR_LICENSE``, ``AVATAX_USE_SANDBOX``, ``AVATAX_COMPANY_NAME``, ``AVATAX_AUTOCOMMIT`` (:ref:`refer to the variables descriptions.<tax_environment_variables>`)
+
+Avalara supports two ways of authentications:
+    1. Through login information (``username`` + ``password``)
+    2. Through the account information (``account_id`` + ``license``)
+
+Use ``AVATAX_USERNAME_OR_ACCOUNT`` and ``AVATAX_PASSWORD_OR_LICENSE`` to pass one of these authentication pairs
+
+Avalara requires a company address to properly calculate taxes.
+The address must be assigned in general section of Saleor's dashboard.
+
+.. warning::
```

Comment:
We've just said in the paragraph above that the company address is required, so this seems unnecessary.